pom.xml

```diff
@@ -1422,6 +1422,7 @@
             <skipIpv6Tests>${skipIpv6Tests}</skipIpv6Tests>
             <proxy.port>${proxy.port}</proxy.port>
             <opennms.version>${project.version}</opennms.version>
+            <org.opennms.rrd.strategyClass>org.opennms.netmgt.rrd.jrobin.JRobinRrdStrategy</org.opennms.rrd.strategyClass>
```

Comment:
Should this be:

```
<org.opennms.rrd.strategyClass>org.opennms.netmgt.rrd.rrdtool.MultithreadedJniRrdStrategy</org.opennms.rrd.strategyClass>

Reply:
All our tests use JRobin. So, the test plugins need this default or all tests will fail.